core/resources/lights_vert.glsl

```diff
@@ -0,0 +1,79 @@
+
+// TODO: 
+//      - BM - port this to STRINGIFY on C++
+
+#ifdef TANGRAM_FRAGMENT_LIGHTS
+    #ifdef TANGRAM_MATERIAL_AMBIENT
+        varying vec4 v_light_accumulator_ambient;
+    #endif
+    #ifdef TANGRAM_MATERIAL_DIFFUSE
+        varying vec4 v_light_accumulator_diffuse;
+    #endif
+    #ifdef TANGRAM_MATERIAL_SPECULAR
+        varying vec4 v_light_accumulator_specular;
+    #endif
+#endif
+
+#ifdef TANGRAM_FRAGMENT_LIGHTS
+void calculateLighting(in vec3 _eyeToPoint, in vec3 _normal ) {
+#else
+vec4 calculateLighting(in vec3 _eyeToPoint, in vec3 _normal ) {
+#endif
+
+    vec3 eye = vec3(0.0, 0.0, 1.0);
```

Comment:
Think we can get rid of the 'eye' vector altogether? I think this value is never changed and all of our positions should be relative to the eye anyway.